.github/workflows/bleeding-edge.yml

```diff
@@ -44,13 +44,15 @@ jobs:
         run: sudo apt-get install libglu1-mesa-dev
       - name: Install build dependencies
         run: |
+          sudo apt-get install swig
```

Comment:
I'm not entirely sure how this will fix the issue because I thought Swig 4.x breaks the agg backend - https://github.com/enthought/enable/issues/360. And IIUC, CI on this PR won't break because this is a cron job.

Can you edit the job schedule to make it run in this PR so that we can verify that this PR fixes the cron job?

Reply:
It works: https://github.com/enthought/enable/runs/1986218229

This is because we restrict the jobs to Ubuntu 18.04 precisely _because of_ #360.